### DISCUSSION TITLE
The role of modern version control systems and dev/ops in open-source statistical computing and reporting systems

### DISCUSSION THREAD

**Discussion by kkmann** [rinpharma/positconf-roundtables-2023#16] [Ideas]
- OS introduces more degrees of freedom/complexity, how to manage?
- Is 'System as code' + VCS the solution?
- What about strong VCS for clinical reporting/scripting/analyses?
- Alternatives to git?
- Taming the complexity and power of git in practice
- DevOps (continuous testing and integration) at scale for clincial reporting

**Comment by epijim** [comment]
I like this - and would consider linking this to the infra, and that interplay between the statistical programmers being deep specialists in a specific way of working (rather than general data scientists), the historical workflows we follow, limits due to integrations (e.g. upstream many sources, downstream medical writers or legacy tools needing specific formats delivered a specific way), and that nagging feeling I think most people involved in clinical reporting SCEs feel wondering what would be possible if we could build a solution from scratch.
